src/main/java/de/picturesafe/search/elasticsearch/config/impl/StandardIndexPresetConfiguration.java

```diff
@@ -185,60 +188,125 @@ public void setUseCompression(boolean useCompression) {
         this.useCompression = useCompression;
     }
 
+    /**
+     * Adds default analysis settings.
+     * @param charMappings  Char mappings für analysis settings
+     */
+    public void addDefaultAnalysisSettings(Map<String, String> charMappings) {
```

Comment:
I'd like naming it addDefaultAnalyzer() better.

Reply:
I chose this name because in the Elasticsearch settings 'analysis' is the parent element for 'analyzer', 'tokenizer', 'char_filter' and 'filter', and this method adds exactly these four aspects.